frameworks/Java/grizzly/src-jersey/main/java/hello/WorldResource.java

```diff
@@ -56,9 +57,11 @@ public Object queries(@QueryParam("queries") String queriesParam) throws Interru
 			Session session = emf.createEntityManager().unwrap(Session.class);
 			session.setDefaultReadOnly(true);
 			try {
-				for (int i = 0; i < queries; i++) {
-					worlds[i] = (World) session.byId(World.class).load(randomWorld());
-				}
+		        //Pick unique random numbers 
+		        final AtomicInteger i = new AtomicInteger(0);
+		        ThreadLocalRandom.current().ints(1, 10000).distinct().limit(queries).forEach(
```

Comment:
It should be 10001 instead of 10000 here and below since [the upper bound is exclusive](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/concurrent/ThreadLocalRandom.html#ints(int,int)).